src/textual/css/_styles_builder.py

```diff
@@ -717,3 +744,9 @@ def process_align_vertical(self, name: str, tokens: list[Token]) -> None:
     process_content_align = process_align
     process_content_align_horizontal = process_align_horizontal
     process_content_align_vertical = process_align_vertical
+
+    def _suggested_property_name_for_rule(self, rule_name: str) -> str | None:
+        possible_matches = get_close_matches(
```

Comment:
Can we factor this function in to a new module. Probably `suggestions.py`. 

I'm thinking two functions, one for a single suggestion, and another for multiple. With signatures along these lines:

```python
def get_suggestion(word:str, possible_words:list[str]) -> str | None:
```

```python
def get_suggestions(word:str possible_words:list[str], count:int) -> list[str]:
```